### DISCUSSION TITLE
Admin customization

### DISCUSSION THREAD

**Discussion by Kavehrafie** [austintoddj/canvas#1053] [Troubleshooting]
I love Canvas, but I want to customize the backend.
What would be the best practice to customize (change the current admin pages and add new admin routes)
Thanks!

**Comment by austintoddj** [comment]
There's not an easy way to do that. If you really wanted to change the backend, you'd need to modify parts of the vendor directory, which isn't ever a good idea. What sorts of things are you looking to include in the admin area?

**Reply by Kavehrafie** [reply]
It would be nice to be able to publish the vue templates of Admin to tweak the design. I would like to add image cropping and new widgets.
Nice app though! Thanks!

**Reply by austintoddj** [reply]
I'll see what I can do about adding that into v7.0

**Comment by carlosjavier10Github** [comment]
First of all thank you very much, I think this project is great, I am a newbie programmer and this project has helped me a lot, just one thing I need to do and I can't, I need to place one more button in the settings screen, that would be below the language selector, I don't know much about vue, and I tried adding one more button row in /vendor/austintoddj/canvas/resources/js/views/EditSettings. vue , then I did "npm run dev" and tried reinstalling with "php artisan canvas:install" and "php artisan canvas:ui" without success(not seeing the button in the browser) , it would be much abuse if you explain me how I can do this? I just need to add a button that will go to an external link (validation for instagram feed), I don't want to modify anything else. 

* or is this not supposed to be done? or can't it be done? since it is not code of my project itself? please excuse my ignorance. 

I hope it is not too much trouble! and again thank you very much! 

sorry if there are mistakes I'm not fluent in English.

**Comment by austintoddj** [comment]
Closing this as it won't be the focus on v7.
